config/default.json

```diff
@@ -112,6 +112,14 @@
         "staticProviderURL": "Your TileServerCache Server Address",
         "geocodingKey":["Your Google Geocoding Key if you Use google as provider"],
         "staticKey":["Your MapQuest or Google Key"],
+        "multistaticmaps": {
```

Comment:
There is also a request to disable static map generation for people who don't want to use them for certain types.
It would be cool if this were:
`
staticmaps: {
  "pokemon": "multi",
  "raid": "standard",
  "invasion": "none"
}
`
or something similar

Reply:
copy that! I'll update branch asap 😃